src/fava/core/ingest.py

```diff
@@ -110,6 +111,8 @@ def load_file(self) -> None:
 
         self.mtime = os.stat(self.module_path).st_mtime_ns
         self.config = mod["CONFIG"]
+        if "HOOKS" in mod:
+            self.hooks = mod["HOOKS"]
```

Comment:
Maybe add a basic check here that this is a list of callables and add an error message otherwise.